CITATION.cff

```diff
@@ -73,6 +73,6 @@ authors:
   orcid: "https://orcid.org/0000-0002-0012-2142"
 title: "JWST Calibration Pipeline"
 version: 1.6.2
```

Comment:
Don't we want/need to update this to 1.7.0 now?

Reply:
oops

Reply:
done